client/src/components/MapComponent.vue

```diff
@@ -1,34 +1,28 @@
 <template>
   <div>
-    <!-- <GmapAutocomplete
-    @place_changed='setPlace'
-    />
-    <button
-    @click='addMarker'
-    >
-    Add
-    </button> -->
-    <br>
     <GmapMap
-      :center='center'
-      :zoom='12'
-      style='width:100%;  height: 1080px;'
+        :center='center'
+        :zoom='12'
+        style='width: 100%;  height: 800px;'
     >
-      <GmapMarker
-        :key="index"
-        v-for="(m, index) in markers"
-        :position="m.position"
-        @click="center=m.position"
-      />
+        <GmapMarker
+            :key="index"
```

Comment:
Using the v-for index just as a key defeats the purpose of having a key. Try using "mark.id" as opposed to the index. If every mark in marker is a unique primitive, just use "mark"